Changes to fee page (#19)
rhinestonewtf/docs: pages/module-sdk/modules/ownable-validator.mdx

import { Tabs } from "nextra/components";

# Ownable Validator

> The Ownable Validator module is a module that allows you to add an ECDSA owners to an account. The owners can then be used to validate transactions and other executions on the account.

Reviewer: ```suggestion
The Ownable Validator module is a module that allows you to add multiple ECDSA owners to an account. The owners can then be used to sign transactions to be executed on the account.
```
